libeos-updater-util/avahi-service-file.c

```diff
@@ -255,8 +255,8 @@ txt_records_to_string (GVariant      *txt_records,
 {
   GVariantIter iter;
   g_autoptr(GString) str = NULL;
-  GVariant *txt_value;
-  const gchar *txt_key;
+  g_autoptr(GVariant) txt_value = NULL;
+  g_autofree gchar *txt_key = NULL;
```

Comment:
No, `txt_key` should remain as `const`, since the GVariant format string used for it is `&s`, not `s`.